gadget-container/Makefile

```diff
@@ -31,6 +31,12 @@ networkpolicyadvisor:
 networkpolicyadvisor/push: networkpolicyadvisor
 	for POD in `kubectl get pod -n kube-system -l k8s-app=gadget -o=jsonpath='{.items[*].metadata.name}'` ; do kubectl cp ./bin/networkpolicyadvisor -n kube-system $$POD:/bin/ ; done
 
+.PHONY: runchookslib
```

Comment:
nit: does it really need to be a PHONY target?

Reply:
I guess not, it could be named `bin/runchooks.so`